OverlayPlugin.Core/ClipboardTechSupport.cs

```diff
@@ -0,0 +1,156 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Windows.Forms;
+using Advanced_Combat_Tracker;
+using Newtonsoft.Json.Linq;
+using RainbowMage.OverlayPlugin.EventSources;
+
+// TODO: print warnings on plugin ordering
+// TODO: get Ravahn to expose more settings and include them
+// TODO: print warning on cactbot plugin / url / user dir mismatch
```

Comment:
I really don't think it makes sense to put the cactbot check here, rather than in the cactbot C# plugin.

My thought is that it's only an issue while the cactbot plugin exists C#-side.

Reply:
Cactbot doesn't haven't any information about overlays or paths.  It could handle user dir mismatch, but this is a minor point that Techno suggested.  I think the 99% case is that the raidboss overlay path is pointing to a different directory than the plugin, and (I think?) cactbot has no information about which overlays exist, and so it sort of needs to be here.